src/ui/qmlcommon/QGCPitchWidget.qml

```diff
@@ -28,15 +28,21 @@ This file is part of the QGROUNDCONTROL project
  */
 
 import QtQuick 2.1
+import QGroundControl.ScreenTools 1.0
 
 Rectangle {
-    property real pitchAngle:      0
-    property real rollAngle:       0
-    property real _reticleHeight:  1
-    property real _reticleSpacing: 17
-    property real _reticleSlot:    _reticleSpacing + _reticleHeight
-    height: 110
-    width:  120
+    property ScreenTools screenTools: ScreenTools { }
+    property real pitchAngle:       0
+    property real rollAngle:        0
+    property real size:             120
+    property real _reticleHeight:   1
+    property real _reticleSpacing:  size * 0.15
+    property real _reticleSlot:     _reticleSpacing + _reticleHeight
+    property real _longDash:        size * 0.40
+    property real _shortDash:       size * 0.25
+    property real _fontSize:        screenTools.dpiAdjustedPointSize(size * 11 / 120);
```

Comment:
Do you need a custom font size here? The default QGC font is 12 pt. If I need larger of smaller fonts I've been using the default point size in the palette and then increment/decrement from there. That way everything follows from the default palette setting.